### PR TITLE
MQE-1443: Removing robo commands and updating Codeception paths

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will...

Update changes to Codeception and remove robo commands.

Closes #MQE-1443

### CODE REVIEW

File: mftf/2.3/introduction.md [hunk 1]
```diff
@@ -128,7 +127,7 @@ The file structure under the both path cases is the same:
 ├── Section
 │   └── ...
 └── Test
-    └── ...
+   └── ...
```
Comment (dshevtsov) [inline mftf/2.3/introduction.md]:
```suggestion
    └── ...
```

File: mftf/2.3/merging.md [hunk 2]
```diff
@@ -294,19 +295,19 @@ Create the `.../FooBackend/Page/BaseBackendPage.xml` file:
 
 ```xml
 <pages ...>
-    <page name="BaseBackendPage" url="admin" area="admin" module="Magento_Backend">
-        <section name="NewExtensionSection"/>
-    </page>
+      <page name="BaseBackendPage" url="admin" area="admin" module="Magento_Backend">
```
Comment (dshevtsov) [inline mftf/2.3/merging.md]: use the same indentation in all XML code snippets

File: mftf/2.3/troubleshooting.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ Troubleshoot your WebDriver issues on various browsers.
 
 You are unable to upload file input using the MFTF actions and are seeing the following exception:
 
-```
-[Facebook\WebDriver\Exception\NoSuchDriverException]
-No active session with ID e56f9260-b366-11e7-966b-db3e6f35d8e1
+```terminal
+   [Facebook\WebDriver\Exception\NoSuchDriverException]
```
Comment (dshevtsov) [inline mftf/2.3/troubleshooting.md]:
```suggestion
[Facebook\WebDriver\Exception\NoSuchDriverException]
```

File: mftf/2.3/troubleshooting.md [hunk 4]
```diff
@@ -18,9 +18,9 @@ Troubleshoot your WebDriver issues on various browsers.
...
+   No active session with ID e56f9260-b366-11e7-966b-db3e6f35d8e1
```
Comment (dshevtsov) [inline mftf/2.3/troubleshooting.md]:
```suggestion
No active session with ID e56f9260-b366-11e7-966b-db3e6f35d8e1
```

Comment (dobooth) [reply mftf/2.3/troubleshooting.md]: Thanks

**Comment by dshevtsov** [comment]
Are these changes applicable to the Magento 2.2 too?

**Comment by dobooth** [comment]
This does not affect 2.2.

**Comment by dobooth** [comment]
running tests

**Pull request merged**